### ISSUE TITLE
DESCRIPTION says GEOS (>= 3.8.0) but 'configure' asserts GEOS (>= 3.4.0)

### ISSUE THREAD

**Issue by HenrikBengtsson** [rspatial/terra#167]
FYI:
The CRAN package page and `DESCRIPTION` file mention GEOS (>= 3.8.0):

https://github.com/rspatial/terra/blob/1e5c86f36038ebe1c9ec797793a631f8f9b063c1/DESCRIPTION#L10

while `configure{.ac}` only looks for (>= 3.4.0):

https://github.com/rspatial/terra/blob/1e5c86f36038ebe1c9ec797793a631f8f9b063c1/configure.ac#L681-L690

**Comment by rhijmans** [comment]
Does this cause problems? I am asking because terra *might* work with GEOS 3.4.0 but I *know* it compiles with 3.8.0 (In fact I regularly test with GEOS 3.6.2 as well, so I could at least lower it to that.) . There is also a discrepancy with GDAL.  Most functionality with GDAL will work with version 2.2.3 --- but not all, and a warning or error is given in those cases. 

So the DESCRIPTION file points to what is desired (known to work well), the configure file to what may really be the minimum requirement. I use the configure from rgdal/sf and they have been around longer and support more backwards compatibility. It is of course easiest not to change too much in that file, so that I can keep using updated versions without much trouble, but I could spend some time in getting this right --- if that is important.

**Comment by HenrikBengtsson** [comment]
I see.  Yes, I think it's confusing to have two different requirements, especially if you're like me a user not in the loop of GEOS but just need to install the package for one reason or the other.  

I would suggest that you have the same minimal GEOS (>= 3.4.0) in both and document the _desired_ GEOS (>= 3.8.0) elsewhere.  If GEOS (>= 3.4.0) is not sufficient, I think you'll eventually get a report on that.

Background: I was running with GEOS 3.5.2 and ran into a compilation error, which surprised me.  Since I saw GEOS (>= 3.8.0), I ended up spending a good hour or so getting GEOS 3.8.1 installed (on CentOS 7) just to find out the error is most likely due to an insufficient PROJ (which I decided to punt on after a couple of hours trying to figure out how to get a modern sqlite3 on this system).

To be clear, I'm not complaining - I now understand what's going on.  I reported because I thought GEOS (>= 3.8.0) was the hard requirement and I wanted to spare others some troubleshooting time, especially those who try to install it and its dependencies from source.
